### ISSUE TITLE
Email formatting from the `pager-me` script

### ISSUE THREAD

**Issue by stephenyeargin** [slackhq/hubot-slack#152]
_Using version 3.2.1_

In PR #125, better email formatting was introduced. However, when using the implementation of [`hubot-pager-me`](https://github.com/hubot-scripts/hubot-pager-me), I see that the script is continuing to emit the `mailto:` link. In reading through both sets of source code, I am not sure where the "bug" really exists.

This is the basic example of the PagerDuty script's implementation:

```
robot.respond /pager(?: me)? as (.*)$/i, (msg) ->
  email = msg.match[1]
  msg.message.user.pagerdutyEmail = email
  msg.send "Okay, I'll remember your PagerDuty email is #{email}"
```

The bot then responds with:

```
user> hubot pager me as stephen@example.com
hubot> Okay, I'll remember your PagerDuty email address is stephen@example.com mailto:stephen@example.com
```

The value of `email` is expanded to `stephen@example.com mailto:stephen@example.com` rather than simply `stephen@example.com`. I'm not sure if that is the payload from Slack that sets that or if something else is going wrong.

---

Related: #125 
Related: https://github.com/hubot-scripts/hubot-pager-me/issues/29

cc/ @paulhammond

**Comment by eirikurn** [comment]
+1 This exact email format issue is affecting my script as well.

**Comment by paulhammond** [comment]
I think the problem here is just that #125 hasn't yet made it into a release. But that's OK, because we just released 3.3.0 which includes the fix!

If I'm wrong please reopen this issue and we'll look into it!

**Comment by stephenyeargin** [comment]
Confirmed fixed.

**Comment by parkr** [comment]
I'm still seeing this. Fixed in 3.3?
